drivers/unix/os_unix.cpp

```diff
@@ -298,7 +298,7 @@ Error OS_Unix::execute(const String &p_path, const List<String> &p_arguments, bo
 		}
 		FILE *f = popen(argss.utf8().get_data(), "r");
 
-		ERR_FAIL_COND_V(!f, ERR_CANT_OPEN);
+		ERR_FAIL_COND_V_MSG(!f, ERR_CANT_OPEN, "Cannot open file '" + String(argss.utf8().get_data()) + "'.");
```

Comment:
This is actually not opening a "file" per se, it's opening a pipe object to read the stream from a process for an executable file with arguments.
`argss` is already a String, so no need to convert it to a C string and back to String.

So something like this:
"Cannot pipe stream from process running with following arguments: " + argss "."